answer/kitamura/JS/13/pulldown.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Document</title>
+</head>
+
+<body>
+  <select name="pull" id="pulldown"></select>
+  <script>
+    const pulldown = document.getElementById('pulldown');
+    const option = [];
+    for (let i = 0; i <= 200; i++) {
+      option[i] = document.createElement('option');
```

Comment:
nits: L15で、配列として定義する必要なさそう?

Reply:
うまくできなかったです、、、

Reply:
イメージとしてはこんな感じを想定してました！（動作確認してないけど動くはず！）

```suggestion
    const pulldown = document.getElementById('pulldown');
    for (let i = 0; i <= 200; i++) {
        const option = document.createElement('option');
        option.setAttribute('value', i);
        option.textContent = i;
        pulldown.appendChild(option);
    }

Reply:
できました！ありがとうございます！おそらく変数の宣言とfor文について知識が足りていないので見直しておきます、、、

Reply:
お！よかったです〜

もともと配列として定義していた、'option'が配列じゃ無くなったため、
setAttribute, textContentとかのプロパティの指定方法が
```diff
- options[i].setAttribute('value', i)
+ option.setAttribute('value', i)
```

のようになってるのも間違えやすいかもしれません〜

Reply:
そうですね、、、まさにそのミスをしていました、、、